### PR TITLE
Update cluster-autoscaler to v2 version 1.0.2

### LINKED ISSUE #858: Update cluster-autoscaler chart to v2 version 1.0.2

### PR DESCRIPTION
**Pull request by knrt10**
Release info:
kubernetes/autoscaler/releases/tag/cluster-autoscaler-chart-1.0.2

Fixes #858

Signed-off-by: knrt10 <kautilya@kinvolk.io>

### CODE REVIEW

**Review by invidian — COMMENTED** [review]
@knrt10 did you test that updated component actually works on Packet? I see a lot of removal in the chart related to Packet, has this been added someplace else? If not, perhaps we should create an issue and submit our changes upstream.

**Comment by knrt10** [comment]
Didn't test it, let me do that

**Review by invidian — APPROVED** [review]
LGTM. We've tested the autoscaler together with @knrt10 and it deploys and runs correctly, but it's functionality is still broken :disappointed:

File: pkg/components/cluster-autoscaler/component.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ const name = "cluster-autoscaler"
 const chartValuesTmpl = `
 cloudProvider: {{ .Provider }}
 image:
-  tag: v1.17.0
+  tag: v1.18.1
```
Comment (invidian) [inline pkg/components/cluster-autoscaler/component.go:39]: Actually, we can remove this override completely, as chart now deploys v1.18.1 by default.

**Comment by knrt10** [comment]
Opened an issue upstrem and awaiting fix from Packet regarding 503 internal server error.

**Comment by invidian** [comment]
@knrt10 the override needs to be removed :)

**Comment by knrt10** [comment]
Aaah, my bad, doing it

**Review by invidian — APPROVED** [review]
LGTM

**Review by ipochi — APPROVED** [review]
LGTM.

**Pull request merged**